x-pack/platform/plugins/private/index_lifecycle_management/public/application/sections/edit_policy/components/phase_icon/phase_icon.scss

```diff
@@ -6,31 +6,71 @@
   align-items: center;
   border-radius: 50%;
   background-color: $euiColorLightestShade;
+
   &--disabled {
     width: $euiSize;
     height: $euiSize;
     margin: $euiSizeS;
   }
+
   &--delete {
     background-color: $euiColorLightShade;
   }
+
   &__inner--hot {
-    fill: $euiColorVis9_behindText;
+    &--borealis {
```

Comment:
It would be nice to implement these changes with emotion instead of adding to the scss. Here's an example of how I did this for upgrade assistant - https://github.com/elastic/kibana/pull/205345/files